src/main/java/com/jootcamp/superboard/board/repository/Entity/BoardNotFoundException.java

```diff
@@ -0,0 +1,15 @@
+package com.jootcamp.superboard.board.repository.Entity;
+
+import com.jootcamp.superboard.common.exception.NotFoundException;
+import com.jootcamp.superboard.common.exception.code.ErrorCode;
+import lombok.Getter;
+
+import static com.jootcamp.superboard.common.exception.code.ErrorCode.NOT_FOUND_BOARD_ERROR;
+
+public class BoardNotFoundException extends NotFoundException {
```

Comment:
board / exception 패키지에 만드셈

```suggestion
public class BoardNotFoundException extends NotFoundException("해당 게시판을 찾을 수 없습니다: " + boardId) {
```